scripts/xtensa-build-zephyr.py

```diff
@@ -323,6 +326,25 @@ def west_init_update():
 	# Do NOT "west update sof"!!
 	execute_command(["west", "update", "zephyr", "hal_xtensa"], timeout=300, cwd=west_top)
 
+def get_sof_version(abs_build_dir):
+	"""[summary] Get version string major.minor.micro of SOF firmware file.
+	When building multiple platforms from the same SOF commit, all platforms share the same version.
+	So for the 1st platform, generate the version string from sof_version.h and later plaforms will
+	reuse it.
+	"""
+	global sof_version
+	if sof_version:
+		return sof_version
+
+	versions = {}
+	with open(pathlib.Path(abs_build_dir, "zephyr/include/generated/sof_versions.h")) as hfile:
```

Comment:
New line

```suggestion
	with open(pathlib.Path(abs_build_dir,
	          "zephyr/include/generated/sof_versions.h")) as hfile:
```

Try `pylint` or `black` (warning: make a git commit or backup before using `black`)